pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -54,8 +54,14 @@ const (
 	// taskRunAgentName defines logging agent name for TaskRun Controller
 	taskRunAgentName = "taskrun-controller"
 	// taskRunControllerName defines name for TaskRun Controller
-	taskRunControllerName = "TaskRun"
-	taskRunNameLabelKey   = "taskrun.knative.dev/taskName"
+	taskRunControllerName  = "TaskRun"
+	taskRunNameLabelKey    = "taskrun.knative.dev/taskName"
+	knativeTaskRunLabelKey = "knative.dev/taskRun"
+	// knativePipelineLabelKey is the label to set to be the name of the Pipeline that the
+	// PipelineRun references
+	knativePipelineLabelKey = "knative.dev/pipeline"
```

Comment:
These `pipeline` and `pipelineRun` constants are duplicated in both controller, can we pull them out in a common file